cypress/support/pages/login_management/pageLoginManagement.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+
+class Login_Management {
+
+  static #passwordRecoveryButton = '[data-testid="forgot-password"]'
+  static #usernameLabel = '[data-testid="username-label"]'
+  static #emailInputBox = '[id="username"]'
+  static #infoMessage = '[data-testid="info-message"]'
+  static #submitButton = '[data-testid="btn-submit"]'
+  static #cancelButton = '[data-testid="btn-cancel"]'
+
+  beingOnLoginPage() {
```

Comment:
`visitLoginPage()` instead of `beingOnLoginPage()`

Reply:
updated!